library_generation/utilities.py

```diff
@@ -407,6 +398,8 @@ def generate_prerequisite_files(
         repo_metadata["rest_documentation"] = library.rest_documentation
     if library.rpc_documentation:
         repo_metadata["rpc_documentation"] = library.rpc_documentation
+    if library.extra_versioned_modules:
+        repo_metadata["extra_versioned_modules"] = library.extra_versioned_modules
```

Comment:
Do you think we should add an unit test to verify this assignment?

Reply:
I added an `extra_versioned_modules` entry in `unit_tests.library_1` and updated the golden `.repo-metadata.json`s. I think that should cover it